Ascend.java

```diff
@@ -1,7 +1,48 @@
+import java.util.Random;
+
 // Generates three integer random numbers in a given range,
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
+		/* lim= limit of random integers
+		creating an arrey with random integers*/ 
+		int lim = Integer.parseInt(args[0]);
+		int[] nums = new int[3];
+		for (int i = 0; i < nums.length; i++) {
+			Random rnd = new Random();
+			int rndN = (int) (rnd.nextInt(1, lim));

```

Comment:
nextInt is a function you haven't seen
nextInt returns an int so there is no need to parse it again to an int...